### ISSUE TITLE
Crashing after some time

### ISSUE THREAD

**Issue by InklingSplasher** [Dragory/modmailbot#299]
Hi, I've been using this bot for quite some time now, and now it keeps crashing after some time.

I'm running this bot with NPM v8.11.3 on Linux Debian 8.11

The error can be found down below:
```
events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: Unhandled MESSAGE_CREATE type: {
  "type": 10,
  "tts": false,
  "timestamp": "2019-06-05T18:02:26.211000+00:00",
  "pinned": false,
  "nonce": null,
  "mentions": [],
  "mention_roles": [],
  "mention_everyone": false,
  "member": {
    "roles": [
      "REDACTED"
    ],
    "premium_since": "2019-06-05T18:02:26.075000+00:00",
    "nick": null,
    "mute": false,
    "joined_at": "2019-06-05T01:26:00.960165+00:00",
    "deaf": false,
    "id": "REDACTED"
  },
  "id": "REDACTED",
  "embeds": [],
  "edited_timestamp": null,
  "content": "",
  "channel_id": "REDACTED",
  "author": {
    "username": "REDACTED",
    "id": "REDACTED",
    "discriminator": "1103",
    "avatar": "REDACTED"
  },
  "attachments": [],
  "guild_id": "REDACTED"
}
    at new Message (/home/nitro/Modmail/modmailbot-master/node_modules/eris/lib/structures/Message.js:103:19)
    at Map.add (/home/nitro/Modmail/modmailbot-master/node_modules/eris/lib/util/Collection.js:41:19)
    at Shard.wsEvent (/home/nitro/Modmail/modmailbot-master/node_modules/eris/lib/gateway/Shard.js:421:65)
    at Shard.onWSMessage (/home/nitro/Modmail/modmailbot-master/node_modules/eris/lib/gateway/Shard.js:1727:26)
    at WebSocket.ws.onmessage (/home/nitro/Modmail/modmailbot-master/node_modules/eris/lib/gateway/Shard.js:1605:33)
    at WebSocket.onMessage (/home/nitro/Modmail/modmailbot-master/node_modules/ws/lib/event-target.js:120:16)
    at emitOne (events.js:116:13)
    at WebSocket.emit (events.js:211:7)
    at Receiver.receiverOnMessage (/home/nitro/Modmail/modmailbot-master/node_modules/ws/lib/websocket.js:719:20)
    at emitOne (events.js:116:13)
    at Receiver.emit (events.js:211:7)
    at Receiver.dataMessage (/home/nitro/Modmail/modmailbot-master/node_modules/ws/lib/receiver.js:414:14)
    at Receiver.getData (/home/nitro/Modmail/modmailbot-master/node_modules/ws/lib/receiver.js:346:17)
    at Receiver.startLoop (/home/nitro/Modmail/modmailbot-master/node_modules/ws/lib/receiver.js:133:22)
    at Receiver._write (/home/nitro/Modmail/modmailbot-master/node_modules/ws/lib/receiver.js:69:10)
    at doWrite (_stream_writable.js:397:12)```

**Comment by evacfk** [comment]
I am getting the same, has crashed two days in a row with the same error.

Running in tmux.

**Comment by Dragory** [comment]
Working on a fix for this. Will push an update soon.

**Comment by Dragory** [comment]
Released version v2.21.3 that should fix this. Please let me know if you still experience crashes.
